skelebot/objects/config.py

```diff
@@ -35,7 +35,8 @@ class Config(SkeleYaml):
         Optional('ports'): And(list, error='\'ports\' must be a List'),
         Optional('components'): And(dict, error='\'components\' must be a Dictionary'),
         Optional('params'): And(list, error='\'params\' must be a List'),
-        Optional('commands'): And(list, error='\'commands\' must be a List')
+        Optional('commands'): And(list, error='\'commands\' must be a List'),
+                Optional('pythonVersion'): And(str, error='\'pythonVersion\' must be a String')
```

Comment:
This looks like it's indented.

Also you can add validation here to avoid having that error msg below.
Something like: 
```suggestion
        Optional('pythonVersion'): And(str, Or(*PYTHON_VERSIONS), error='\'pythonVersion\' must be one of:' + ', '.join(PYTHON_VERSIONS)
```

Reply:
I was thinking if people provide a wrong pythonVersion, they'll be using the default version. Is it better to be strict?

Reply:
well, my thinking was that if they put in a wrong version and skelebot auto-corrects, they not be aware of it (e.g. they miss the warning message), which could cause confusion downstream.